src/Payum/Paypal/ExpressCheckout/Nvp/Action/CancelAction.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Payum\Paypal\ExpressCheckout\Nvp\Action;
+
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\Request\Cancel;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Paypal\ExpressCheckout\Nvp\Api;
+use Payum\Paypal\ExpressCheckout\Nvp\Request\Api\DoVoid;
+
+class CancelAction extends PurchaseAction
```

Comment:
Lazy copy of CaptureAction?  Fixed to extend GatewayAwareAction and execute a Sync after executing a DoVoid on all payment requests that are Authroized.